### PR TITLE
Lead Links can revoke role assignments

### PR DESCRIPTION
**Pull request by kraenhansen**
This pull request is based on a topic on the community of practice: http://community.holacracy.org/topic/can-lead-links-unassign-individuals-from-roles
It seems to me that the constitution is not explicit on the fact that the Lead Link can in fact revoke the assignment of a role to an individual. This causes confusion - as the LL role description seems in conflict with the 2.4 section of the constitution itself.

I suggest four changes to the constitution:
1.  Adding "and revoke these assignments", because this is the dominant interpretation (judged by the discussion on the community) and the way the LL role is defined: It has domain over "Role assignments within the Circle".
2.  Adding "at any time" to stress that the LL (per default) does not need to seek consensus or approval from anyone or any process.
3.  Changing to a plural "authorities" as it is not just the authority to assign nor the authority to revoke assignments, that might be limited or delegated.
4.  Changing "people" to "Partners" as this is used instead of "people" or "individuals" throughout the constitution, and I imagine that a LL would not like to assign a role to an individual that does not "agree to take part in the governance and operations of the Organization" (from the definition of a Partner).

Side-note: I would actually like if the wording around assigning people to roles were "invite to roles", at least this is how we interpret the constitution, as in fact the person being assigned can resign at any given time and we like the Lead Link to go through the role description to make sure the person understrands the accountabilities they are taking upon themselves. But this might be an entirely different pull-request :)

### CODE REVIEW

**Comment by kraenhansen** [comment]
By the way - I was thinking about using the word "unassign" but found this was not a real word :)

Could have picked "dissociate" as http://english.stackexchange.com/questions/59463/antonym-to-assign suggests, but I went for "revoke" as http://english.stackexchange.com/questions/156986/if-i-assign-someone-a-task-what-is-the-correct-verb-for-unassigning-them suggested.

**Comment by smichel17** [comment]
If you split this into two changes, one for "people" => "Partners" and the other with the revoke change, the first could probably be merged with no discussion (in my opinion as a random community member).

File: Holacracy-Constitution.md [hunk 1]
```diff
@@ -142,15 +142,15 @@ The Lead Link of a Circle may specially appoint additional persons to serve as C
 
 ### 2.4 Role Assignment
 
-The Lead Link of a Circle may assign people to fill Defined Roles in the Circle, unless that authority has been limited or delegated.
+The Lead Link of a Circle may assign Partners to fill Defined Roles in the Circle and revoke these assignments at any time, unless these authorities has been limited or delegated.
```
Comment (abresas) [inline Holacracy-Constitution.md:145]: You probably meant "these authoritites have" not "these authorities has" here

**Comment by brianjrobertson** [comment]
The change from people to Partners is a significant change, and needs tension-driven justification on why that limit is needed (currently this clause is used by some organizations to assign non-Partners to roles).

**Comment by brianjrobertson** [comment]
I suggest breaking out the other change into its own pull request, separate from that one...

**Comment by kraenhansen** [comment]
That totally makes sense - thanks :) I mistakenly thought it was mistake.

With this fact, would you expect the same of a person filing a role that you would of a Partner filing the same role? I am asking because there are mentions of Partner in the "1.2 Responsibilities of Role-Filling", "1.3 Authority to Act", "1.4 Authority Over Domains", "2.3.1 Base Membership" and other places, where it might make sense to not differentiate. A circle member might be confused that the Duties of Circle Members hold for non-partners but the other sections doesn't. Is this because a Holacratic organisation wouldn't expect every (non-Partner) role-filling person to have read the constitution?

**Comment by kraenhansen** [comment]
I've created a new - simpler pull request :) https://github.com/holacracyone/Holacracy-Constitution/pull/107

**Comment by brianjrobertson** [comment]
You wouldn't necessarily - depends on your contractual relationship with the person...

**Pull request closed without merging**